### PR TITLE
fix: use constants not magic numbers

### PR DESCRIPTION
**Pull request by kuisathaverat**
# Description

Minor refactor replace magic numbers with constants. They will appear in the Java docs making it easier to know which value is for what.

A brief summary describing the changes in this pull request. See 
[JENKINS-64718](https://issues.jenkins-ci.org/browse/JENKINS-64718) for further information. 

# Submitter checklist
- [X] Link to JIRA ticket in description, if appropriate.
- [X] Change is code complete and matches issue description
- ~[ ] Automated tests have been added to exercise the changes~
- ~[ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.~

# Reviewer checklist
- [ ] Run the changes and verify that the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

# Documentation changes
- ~[ ] Link to jenkins.io PR, or an explanation for why no doc changes are needed~

# Users/aliases to notify

@timja @bitwiseman

### CODE REVIEW

**Review by bitwiseman — APPROVED** [review]
I feel like there's a second PR here to convert these to enums, but this PR is simple, direct, and completely safe.

File: src/main/java/org/jenkinsci/plugins/github_branch_source/BranchDiscoveryTrait.java [hunk 1]
```diff
@@ -52,6 +52,23 @@
...
+    */
+    public static final int NONE = 0;
+    /**
+     * Exclude branches that are also filed as PRs.
+    */
+    public static final int EXCLUDE_PRS = 1;
+    /**
+     * Only branches that are also filed as PRs.
+    */
+    public static final int ONLYP_RS = 2;
```
Comment (timja) [inline src/main/java/org/jenkinsci/plugins/github_branch_source/BranchDiscoveryTrait.java:66]: shouldn't this be `ONLY_PRS`?

Comment (kuisathaverat) [reply src/main/java/org/jenkinsci/plugins/github_branch_source/BranchDiscoveryTrait.java:66]: 🤦‍♂️ https://github.com/jenkinsci/github-branch-source-plugin/pull/377

Comment (bitwiseman) [reply src/main/java/org/jenkinsci/plugins/github_branch_source/BranchDiscoveryTrait.java:66]: My bad, I should have caught that.

**Pull request merged**